src/data/detail-page/tabs/backend.ts

```diff
@@ -0,0 +1,3 @@
+import { categoryId } from './tabsInterfaces';
+
+export const backend: categoryId[] = ['learn', 'jobs', 'resources'];
```

Comment:
Types and interfaces names should be capitalized by convention: https://google.github.io/styleguide/tsguide.html#identifiers 

Let's rename our type to `CategoryId` :)

Reply:
Done